test/MemeBaseLaunchCampaign.js

```diff
@@ -61,18 +58,91 @@ const main = async () => {
         accounts, amounts);
     await memeBase.deployed();
 
-    // Summon a new meme token
-    await memeBase.summonThisMeme(name, symbol, totalSupply, {value: smallDeposit});
+    // Summon a new meme token - negative cases
```

Comment:
Added lots more tests. I'm now more confident that the issue stems from the uniswap logic entirely.

Basically, we can see that occasionally line 131 raises now. Sometimes we also pass it. It should predictably pass. The fact that it only raises now with the event emission check suggests that there's some sort of "silent" failure inside the unleash logic..